doc/user/load_save.md

```diff
@@ -813,6 +813,43 @@ std::cout << std::endl << std::endl;
 mlpack::data::Save("mlpack-favicon.png", image, info, true);
 ```
 
+### Letterbox transform
+
+The letterbox transform resizes an image's dimensions to `imgSize x imgSize` but
+keeps the aspect ratio of the original image. Whitespace is then filled in
+with `fillValue`.
+
+- `LetterboxImage(src, opt, imgSize, fillValue)`
+  * `src` is a [column-major matrix](matrices.md) containing a single image,
+    where the image is represented as a flattened vector in one column.
+  * `opt` is a [`data::imageOptions&`](#dataimageinfo) containing info on
+    the dimensions of the image.
+  * `imgSize` is a `const size_t` determining the new width and height of
+    `src`.
+  * `fillValue` is the white space value that pads out the resized image.
+    Each channel will be filled in with this value, i.e if `fillValue` is 127
+    then each RGB channel will be 127. Only images with 1 or 3 channels
+    can be used.
```

Comment:
```suggestion
    then each RGB channel will be 127.
  * Only images with 1 or 3 channels can be used.
```
Seems like it would be better as a separate bullet point?